### PR TITLE
instructorFeedbackSessions: mention that archived sessions are not shown #2861

### LINKED ISSUE #2861: instructorFeedbackSessions: mention that archived sessions are not shown
Users might get worried if they don't see those sessions.
It can be a simple message under the sessions table.

### PR DESCRIPTION
**Pull request by drunken-saint**
fixes #2861

### CODE REVIEW

**Comment by kanghj** [comment]
Good work figuring out how to modify the tests. The previously failing tests, AdminHomePageUiTest and InstructorFeedbackEditPageUiTest, have been fixed on master. Can you sync with our repo and merge master into this branch ensure that the tests still pass after your changes? 

Can you also post a screenshot of the UI change here, so that we can have a look at changes again?

File: src/main/webapp/jsp/instructorFeedbacks.jsp [hunk 1]
```diff
@@ -696,6 +696,7 @@
                 }
             %>
         </table>
+        <p align-center>Note: The table above doesn't contain sessions from archived courses. To view sessions from an archived course, unarchive the course first.</p>
```
Comment (kanghj) [inline src/main/webapp/jsp/instructorFeedbacks.jsp]: I'm not sure if there's an align-center html attribute. In your previous pull request, I mentioned to use the align-center *css class*.

**Comment by drunken-saint** [comment]
screenshot 74
My text editor suggest align-center when I was trying to replace align="center"  when you said that the latter has some issues with it . I am attaching a screenshot of the sessions page to get you a view . Tell me what you think of it . And I've synced the branch with the master and will update about the test results later .

**Comment by damithc** [comment]
I prefer to use grey color text for this message.

**Comment by drunken-saint** [comment]
Yeah , @damithc  grey made it better . Here is the screenshot . I am modifying the test files now . Will update you guys soon.
screenshot 76

**Comment by drunken-saint** [comment]
I've modified the test files now . Issue2861 is devgreen . Looks good to merge .

**Comment by kanghj** [comment]
The reason why I was asking about the align-center was that you were trying to center the text in the previous pull request. 

Nevertheless, this looks good to me.

**Pull request merged**